### ISSUE TITLE
error on my own data

### ISSUE THREAD

**Issue by luoshuiyue** [lilygoli/SpotLessSplats#8]
when I run the command `python spotless_trainer.py  --data_dir [data directory] --data_factor 8   --result_dir [result directory] --loss_type robust --semantics --no-cluster --train_keyword "clutter" --test_keyword "extra"` , I encounter this error:
![image](https://github.com/user-attachments/assets/15d0c8d1-42c9-4664-bb99-8e0844f2340a)
I want to ask how to arrange my dataset to reproduce the desired results:
this is my dataset:
![image](https://github.com/user-attachments/assets/6eea75c4-d3b2-4a56-ac4d-1061b0cd8695)
![image](https://github.com/user-attachments/assets/b78fcd66-fd95-474a-bc38-5a40c013d3b6)
![image](https://github.com/user-attachments/assets/dc119baf-c830-469c-aeb8-6fc9ff08c742)

**Comment by lilygoli** [comment]
yes I am uploading the dataset folders to be shared. Please follow the thread in issue https://github.com/lilygoli/SpotLessSplats/issues/11 for updates.
